javalib/src/main/scala/java/util/AbstractSet.scala

```diff
@@ -0,0 +1,46 @@
+package java.util
+
+import scala.annotation.tailrec
+
+abstract class AbstractSet[E]() extends java.util.AbstractCollection[E] with java.util.Set[E] {
+	override def equals(o: Any) = {
```

Comment:
Always write the result type of public methods explicitly, even when they are overridden.